docs/api/saved-objects/copy_to_space.asciidoc

```diff
@@ -0,0 +1,290 @@
+[role="xpack"]
+[[saved-objects-api-copy-to-space]]
+=== Copy to Space
+++++
+<titleabbrev>Copy to Space</titleabbrev>
+++++
+
+experimental[This functionality is *experimental* and may be changed or removed completely in a future release.]
+
+Copies saved objects from one space to other spaces.
+
+Note: You cannot access this endpoint via the Console in Kibana.
```

Comment:
Thoughts on this? We've tried to include this for all Kibana API endpoints, because a lot of users are accustomed to using the Kibana Console (dev tools) to execute these API requests.

Reply:
I would like to remove this note from all of the individual pages, and let it lively solely on the [main API page](https://www.elastic.co/guide/en/kibana/current/api.html). What do you think?

Reply:
I agree with @KOTungseth.